dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -144,6 +144,12 @@ void CommandBarFlyoutCommandBar::OnApplyTemplate()
         {
             moreButton.IsTabStop(false);
         }
+        m_moreButtonClicked = moreButton.Click(winrt::auto_revoke, { this, &CommandBarFlyoutCommandBar::MoreButtonClicked });
```

Comment:
>Click [](start = 41, length = 5)

Hitting tab after opening the CommandBarFlyout also opens the Secondary commands